theano/tensor/tests/test_sharedvar.py

```diff
@@ -27,6 +27,7 @@ def makeSharedTester(shared_constructor_,
                      theano_fct_,
                      ref_fct_,
                      cast_value_=np.asarray,
+                     need_decorator=True,
```

Comment:
That is not really an informative name...
Maybe `expect_fail_fast_shape_inplace=True` for instance.